ext/src/ruby_api/memory_type.rs

```diff
@@ -2,13 +2,19 @@ use super::root;
 use magnus::{function, method, scan_args, Error, Module as _, Object, Value};
 use wasmtime::MemoryType as MemoryTypeImpl;
 
+/// @yard
+/// @see https://docs.rs/wasmtime/latest/wasmtime/struct.MemoryType.html Wasmtime's Rust doc
 #[derive(Clone, Debug)]
 #[magnus::wrap(class = "Wasmtime::MemoryType")]
 pub struct MemoryType {
     inner: MemoryTypeImpl,
 }
 
 impl MemoryType {
```

Comment:
Same thing here. I'd try to make this a private class, and expose the relevant information/API on the `Memory` class instead.